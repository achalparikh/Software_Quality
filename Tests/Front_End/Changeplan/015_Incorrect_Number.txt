login
admin
changeplan
John Doe
00013
logout
quit