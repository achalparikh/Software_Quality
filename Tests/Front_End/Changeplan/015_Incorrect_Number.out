Login to start:
Enter type of account: "standard" or "admin"
You are now logged in as "John Doe"
Enter a command:
Enter an account holder:
Enter an account number belonging to the account holder:
Error, account number 00013 does not match any accounts owned by "John Doe"
Enter a command:
You are now logged out